index.html

```diff
@@ -5,12 +5,74 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="icon" href="cup-cake-icon.png" type="image/icon type">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
     <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+    <link rel="stylesheet" href="style.css">
 </head>
 
 <body>
-    <!-- Add your markup here -->
+    <header>
+        <div class="logo">
+            <img src="/cake-logo.png" alt="logo">
+        </div>
+        <nav>
+            <ul>
+                <input type='checkbox' id='check' />
+                <div class="menu">
+                <li><a href="#">Home</a></li>
+                <li><a href="#">Cakes</a></li>
+                <li><a href="#">Orderin</a></li>
+                <li><a href="#">Lessons</a></li>
+                <li><a href="#">About</a></li>
+                </div>
+                <label for="check" class="open-menu"><i class="fa-solid fa-bars"></i></label>
+            </ul>
+        </nav>
+        <p>The best cakes in town delivered to your door</p>
+    </header>
+
+    <main>
+        <section class="navigation">
+            <ul>
+                <li><a href="#">Home</a></li>
+                <li><a href="#">Cakes</a></li>
+                <li><a href="#">Orderin</a></li>
+                <li><a href="#">Lessons</a></li>
+                <li><a href="#">About</a></li>
+            </ul>
+        </section>
+        <section class="container">
+            <h1>Welcome</h1>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Earum nemo quae dolores doloremque officiis voluptatem tenetur impedit, reiciendis inventore est eligendi perferendis, eum harum itaque at natus blanditiis nam velit!</p>
+        </section>
+
+        <section class="main-pic">
+            
+        </section>
+
+        <section class="collage">
```

Comment:
Good job on following the mobile first design! For the desktop the images in the collage looks slightly squished probably due to different image height variance and well thought that you are trying to align it.
**Improvements 3:** adjust the design so that your desktop viewport don't have squished images.